index.bs

```diff
@@ -2365,48 +2365,99 @@ Reviews and Review Responsibilities</h4>
-	that the group has solicited wide review.
-
-	A [=Working Group=] could present evidence that wide review has been received,
-	irrespective of solicitation.
-	But it is important to note that receiving many detailed reviews
-	is not necessarily the same as wide review,
-	since they might only represent comment
-	from a small segment of the relevant stakeholder community.
+	Wide review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, to allow different parts of the web community to iterate
+	on requirements, problems, and solutions. 
+
+	When a [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>. 
+
+	As a specification progresses along the Recommendation Track, Working Groups
+	<em class="rfc2119">should</em> solicit review from some or all of the following
+	stakeholders: 
+
+	<ul>
+		<li>The Working Group responsible for the specification</li>
+		<li>The horizontal review groups (see 6.2.3.2)</li>
+		<li>The Working Groups and organisations documented in the Working Group's charter</li>
+		<li>The general public</li>
+		<li>Other relevant standards organizations</li>
+	</ul>
+
+	The Director <em class="rfc2119">must</em> consider evidence of wide review before
+	approving transitions. Appropriate evidence may include: 
+
+	<ul>
+		<li>Documented requests for wide review</li>
+		<li>Documented discussion and resolution of issues raised during wide review</li>
+	</ul>
+
+<h5 id="horizontal-review">
+Horizontal Review</h5>
+
+	The objective of <dfn id=dfn-horizontal-review export>horizontal review</dfn> is
+	to make sure specifications are robust in terms of accessibility, internationalisation,
+	privacy, security, and technical architecture by the time they become a [=W3C Recommendation=]. 
+
+	Horizontal review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, so that accessibility, internationalisation, privacy,
+	security, and technical architecture are considered throughout. 
+
+	Before a proposed specification transitions onto the Recommendation Track, the
+	group responsible for the specification <em class="rfc2119">must</em> solicit a
+	horizontal review from each of the horizontal review groups (in many cases,
+	submitting a self-review to the reviewing group will satisfy this requirement).
+
+	The <dfn id=dfn-horizontal-review-groups export>Horizontal Review Groups</dfn> are:
+
+	<ul>
+		<li><a href="https://www.w3.org/WAI/APA/">Accessible Platform Architectures Working Group</a></li>
+		<li><a href="https://www.w3.org/International/core/Overview">Internationalisation Working Group</a></li>
+		<li><a href="https://www.w3.org/Privacy/">Privacy Interest Group</a></li>
+		<li><a href="https://www.w3.org/2001/tag/">Technical Architecture Group</a></li>
+	</ul>
```

Comment:
Isn't there a Security group too? I find this confusing since the TAG seems to require a security self-review too. But @plehegar has recently shared slides that do list Security explicitly.